samples/ui-library/src/components/UILibraryComponent/FormControls.tsx

```diff
@@ -0,0 +1,361 @@
+import Checkbox from "@geocortex/web/ui/checkbox";
+import FormControl from "@geocortex/web/ui/form-control";
+import FormControlLabel from "@geocortex/web/ui/form-control-label";
+import FormHelperText from "@geocortex/web/ui/form-helper-text";
+import FormLabel from "@geocortex/web/ui/form-label";
+import Input from "@geocortex/web/ui/input";
+import MenuItem from "@geocortex/web/ui/menu-item";
+import Radio from "@geocortex/web/ui/radio";
+import RadioGroup from "@geocortex/web/ui/radio-group";
+import Select from "@geocortex/web/ui/select";
+import Typography from "@geocortex/web/ui/typography";
+import React from "react";
+
+function CheckboxDemo() {
+    const [state, setState] = React.useState({
+        gilad: true,
+        jason: false,
+        antoine: false,
+    });
+
+    const { gilad, jason, antoine } = state;
+
+    const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+        setState({ ...state, [event.target.name]: event.target.checked });
+    };
+
+    return (
+        <>
+            <FormControlLabel label="Checkbox" control={<Checkbox />} />
+            <div>
+                <FormControl {...{ component: "fieldset" }}>
+                    <FormLabel {...{ component: "legend" }}>
+                        Assign responsibility
+                    </FormLabel>
+                    <FormControlLabel
+                        control={
+                            <Checkbox
+                                checked={gilad}
+                                onChange={handleChange}
+                                name="gilad"
+                            />
+                        }
+                        label="Gilad Gray"
+                    />
+                    <FormControlLabel
+                        control={
+                            <Checkbox
+                                checked={jason}
+                                onChange={handleChange}
+                                name="jason"
+                            />
+                        }
+                        label="Jason Killian"
+                    />
+                    <FormControlLabel
+                        control={
+                            <Checkbox
+                                checked={antoine}
+                                onChange={handleChange}
+                                name="antoine"
+                            />
+                        }
+                        label="Antoine Llorca"
+                    />
+                </FormControl>
+            </div>
+        </>
+    );
+}
+
+function InputDemo() {
+    const [value, setValue] = React.useState("");
+
+    const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+        setValue((event.target as HTMLInputElement).value);
+    };
+
+    return (
+        <>
+            <div>
```

Comment:
What would you think of having a`withStyles()` override of FormControl that sets `display: flex` (overriding `inline-flex`)? Or even just some child selector CSS on the container? Then you wouldn't need to wrap every form control in a div.

Reply:
Turn out to be a bit of a mess to remove the `div` elements. Left as is.